webknossos/examples/write_images_to_harddrive.py

```diff
@@ -0,0 +1,22 @@
+from pathlib import Path
+
+from webknossos import Dataset
+
+INPUT_FOLDER = Path("path/to/image_files")
+OUTPUT_FOLDER = Path("path/to/webknossos_organization")
+
+
+def main() -> None:
+    """Convert a folder of image files to a wkw dataset."""
+    Dataset.from_images(
+        input_path=INPUT_FOLDER,
+        output_path=OUTPUT_FOLDER,
```

Comment:
Output is likely only needed when you have phyiscal access to the file system of a WK instances. This is rarely the case. I would prefer to aim for users who will upload the data.

Reply:
It seems to me as if an output_path is a mandatory argument of the from_images method.